get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc

```diff
@@ -0,0 +1,147 @@
+
+== Navigation
+
+The menu along the left side of Neo4j Desktop shows high-level icons for projects, settings, your profile, and Neo4j info.
+At the top, the folder with a bookmark (image:{img}projects_icon.png[width=20]) shows the list of projects you have created.
+If this is the first time using Neo4j for desktop, the default `My Project` will be the only one.
+You can either create a new project to work with your use case or create your database within the default project, but this guide will use the default `My Project`.
+
+image::{img}desktop_mainscreen.png[]
+
+=== Creating and Deleting a Project
+
+To create a new project, simply click `+ New` at the right of the `Projects` menu.
+The app will add a row in the menu for a project named `Project`. You can change the name by clicking on the pencil icon that appears when you hover on the project name in the right pane.
+To remove the project, hover over the `Project` row in the projects menu and click the recycle can icon that appears on the right side (image:{img}delete_project_icon.png[width=20]).
+
+=== Settings
+
+Application settings can be modified or viewed by clicking the gear towards the bottom of the left-side menu in Neo4j for desktop (image:{img}settings_icon.png[width=20]).
+You can modify any of these settings at any time.
+
+The first block is for privacy settings where you can adjust what data is sent back to Neo4j for assistance and future improvements.
+Under `Neo4j Distributions`, it shows which versions of the database are used in your projects (if any) and lists the currently set up projects and databases.
+
+The last section on `Developer Tools` is to add and integrate graph apps with Neo4j.
+You can also access the javascript console of an application for debugging and error reporting through the developer tools.
+
+=== Profile and About
+
+The person icon (image:{img}profile_icon.png[width=20]) shows the profile you created when you signed into Desktop and allows you to sign out of the current account and into another one.
+It shows who is currently logged into Neo4j for desktop.
+
+The bottom icon (image:{img}neo4j_icon.png[width=20]) is information about Neo4j itself and includes links to the main company website, as well as release and license information.
+
+== Creating a graph database
+
+From the Projects icon on the left-side menu, choose (or create) a project.
+A panel will display on the right to show the databases in your project and the plugins available or installed.
+
+To create a database, click in the dotted square that says `New Graph`.
+Then click the kbd:[Create a Local Graph] button.
+Next, you will type in a name for your database and set a password and choose a version (default is latest version), then click kbd:[Create].
+It may take a few minutes to download the version and actually create the database.
```

Comment:
mention that it shows with a little icon which versions have been downloaded already